valid-palindrome/bhyun-kim.py

```diff
@@ -0,0 +1,35 @@
+"""
+Solution:
+
+Algorithm: 
+    1. Convert the string to lowercase.
+    2. Remove all non-alphanumeric characters.
+    3. Check if the string is equal to its reverse.
+
+Time complexity: O(n)
+Space complexity: O(n) 
+"""
+
+
+class Solution:
+    def isPalindrome(self, s: str) -> bool:
+        s = s.lower()
+        s = list([c for c in s
+            if c.isalpha() or c.isnumeric()])
```

Comment:
이미 리스트인데, `list()` 함수로 감싸는 것은 불필요한 것 같습니다.

```suggestion
        s = [c for c in s
            if c.isalpha() or c.isnumeric()]
```